### DISCUSSION TITLE
Repos for the gallery examples?

### DISCUSSION THREAD

**Discussion by PatrikN-work** [eclipse-glsp/glsp#1214] [Q&A]
Is is possible to access the code /repos for the examples in the gallery? https://eclipse.dev/glsp/gallery/

I think it would help to udnerstand things that are not documented yet.

**Comment by planger** [comment]
Hi Patrik,

good idea! However, many of the screenshots of the gallery are from proprietary closed-source application of GLSP adopters, who were kind enough to share a glance of their tools. So for many of them, we can't point to any public source code.

The ones that are open source:
* [Workflow editor](https://github.com/eclipse-glsp/glsp-examples/tree/master/workflow)
* [BIG UML tool](https://github.com/borkdominik/bigUML)
* [Open BPMN](https://github.com/imixs/open-bpmn)
* [Ecore GLSP](https://github.com/eclipse-emfcloud/ecore-glsp)

I'll put it on our todo list to add links to the webpage too.

**Reply by planger** [reply]
https://github.com/eclipse-glsp/glsp/issues/1215

**Comment by PatrikN-work** [comment]
Thanks Philip!

What I was after, even though buig examples like the ones you added are good, is small ones to showcase a certain aspect, like layout , undo-redo, nagigation etc.

**Reply by planger** [reply]
I see, unfortunately those screenshots are mostly taken from "larger tools". I think for the smaller aspects, the best resource is the workflow example which covers some of the aspects (like navigation and layouting). Undo/redo is also covered by the project templates.
